app/helpers/vaccines_helper.rb

```diff
@@ -0,0 +1,5 @@
+module VaccinesHelper
+  def vaccine_heading(vaccine)
+    sprintf("%s (%s)", vaccine.brand, t(vaccine.type, scope: "vaccines"))
```

Comment:
Yeah, `sprintf` is clearer than string interpolation isn't it 👍

Reply:
In this case. I wanted to do `"%s (%s)" % [vaccine.brand, t(vaccine.type, scope: "vaccines")]" but Rubocop got pushy about it ... I think it has a chip on it's shoulder about looking too much like Python.